### PR TITLE
[Docs] Japanese translation of docs/getting_started_docker.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/getting_started_docker.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/getting_started_docker.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Docker クイックスタート
+
+<!---
+  original document: 0.9.32:docs/getting_started_docker.md
+  git diff 0.9.32 HEAD -- docs/getting_started_docker.md | cat
+-->
+
+このプロジェクトは、プライマリオペレーティングシステムに大きな変更を加えることなくキーボードの新しいファームウェアを非常に簡単に構築することができる Docker ワークフローを含みます。これは、あなたがプロジェクトをクローンしビルドを実行した時に、他の人とまったく同じ環境と QMK ビルドインフラストラクチャを持つことも保証します。これにより、人々はあなたが遭遇した問題の解決をより簡単に行えるようになります。
```
Comment (shelaf) [inline docs/ja/getting_started_docker.md:8]:
```suggestion
このプロジェクトは、プライマリオペレーティングシステムに大きな変更を加えることなくキーボードの新しいファームウェアを非常に簡単に構築することができる Docker ワークフローを含みます。これは、あなたがプロジェクトをクローンしビルドを実行した時に、他の人とまったく同じ環境と QMK ビルド基盤を持つことも保証します。これにより、人々はあなたが遭遇した問題の解決をより簡単に行えるようになります。
```

File: docs/ja/getting_started_docker.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## 必要事項
+
+主に必要なものは動作している `docker` のインストールです。
+* [Docker CE](https://docs.docker.com/install/#supported-platforms)
+
+## Usage
```
Comment (shelaf) [inline docs/ja/getting_started_docker.md:15]:
```suggestion
## 使い方
```

File: docs/ja/getting_started_docker.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+```bash
+git clone --recurse-submodules https://github.com/qmk/qmk_firmware.git
+cd qmk_firmware
+```
+
+キーマップをビルドするために以下のコマンドを実行します:
+```bash
+util/docker_build.sh <keyboard>:<keymap>
+# For example: util/docker_build.sh planck/rev6:default
```
Comment (shelaf) [inline docs/ja/getting_started_docker.md:27]:
```suggestion
# 例えば: util/docker_build.sh planck/rev6:default
```

File: docs/ja/getting_started_docker.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+<!---
+  original document: 0.9.32:docs/getting_started_docker.md
+  git diff 0.9.32 HEAD -- docs/getting_started_docker.md | cat
+-->
+
+このプロジェクトは、プライマリオペレーティングシステムに大きな変更を加えることなくキーボードの新しいファームウェアを非常に簡単に構築することができる Docker ワークフローを含みます。これは、あなたがプロジェクトをクローンしビルドを実行した時に、他の人とまったく同じ環境と QMK ビルドインフラストラクチャを持つことも保証します。これにより、人々はあなたが遭遇した問題の解決をより簡単に行えるようになります。
+
+## 必要事項
+
+主に必要なものは動作している `docker` のインストールです。
```
Comment (shelaf) [inline docs/ja/getting_started_docker.md:12]:
```suggestion
主な前提条件は動作する `docker` がインストールされていることです。
```

File: docs/ja/getting_started_docker.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/getting_started_docker.md]: ご指摘ありがとうございます。
最初はビルドインフラと訳していて、これはさすがに訳になってないなとそのままにカタカナにしてしまっていました。

File: docs/ja/getting_started_docker.md (hunk 4 above)
Comment (umi-umi) [reply docs/ja/getting_started_docker.md]: ご指摘ありがとうございます。

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**